eth/handler_eth.go

```diff
@@ -64,24 +64,21 @@ func (h *ethHandler) AcceptTxs() bool {
 func (h *ethHandler) Handle(peer *eth.Peer, packet eth.Packet) error {
 	// Consume any broadcasts and announces, forwarding the rest to the downloader
 	switch packet := packet.(type) {
-	case *eth.BlockHeadersPacket:
-		return h.handleHeaders(peer, *packet)
+	case *eth.BlockHeadersPacket: //  TODO(karalabe): Delete before PR merge
+		log.Error("Old header handler still got called")
+		return errors.New("old header handler still got called")
 
-	case *eth.BlockBodiesPacket:
-		txset, uncleset := packet.Unpack()
-		return h.handleBodies(peer, txset, uncleset)
+	case *eth.BlockBodiesPacket: //  TODO(karalabe): Delete before PR merge
```

Comment:
Here's still a todo

Reply:
Yes, I'll delete this after a few benchmark syncs. Just want to make sue these don't end up in live code.